satpy/readers/slstr_l1b.py

```diff
@@ -133,15 +178,17 @@ def get_dataset(self, key, info):
             variable = self.nc['{}_BT_{}{}'.format(self.channel, self.stripe, self.view[0])]
         else:
             variable = self.nc['{}_radiance_{}{}'.format(self.channel, self.stripe, self.view[0])]
+        print(np.nanmean(variable))
+        radiances = self._apply_radiance_adjustment(variable)
+        print(np.nanmean(radiances))
```

Comment:
You plan on removing the print statements, right?

Reply:
Aah, you found my deliberate test to see how well you were reading the code ;-)
Oops - forgot to push the commit where I remove those lines. Thanks!